examples/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Compile C++ example
```

Comment:
You should move each example to its own folder (_e.g._ `CXX_host`) and put a `CMakeLists.txt` for a standalone project.